tests/client-sdk/inference/test_inference.py

```diff
@@ -371,6 +378,33 @@ def test_image_chat_completion_streaming(llama_stack_client, vision_model_id):
     assert any(expected in streamed_content for expected in {"dog", "puppy", "pup"})
 
 
+def test_image_chat_completion_base64_data(
```

Comment:
we can combine these into a same test code with two parameterized fixtures?

Reply:
it's a pita to use fixtures in a parameterize declaration. instead i parameterize the type and select the input. ok?

Reply:
sure